example/policy_example.py

```diff
@@ -92,11 +95,14 @@ def policy_based_train(env, agent, memory, args):
                 memory.update_priorities(experience['indices'], info)
 
```

Comment:
We also want to record data at every step within the episode.

We recently updated the `agent.train_policy` method to return a dictionary of useful information (actor/critic losses), including these in the log would be good as it means we can also plot them if need be.

Reply:
resolved in commit a91da49.